production.Dockerfile

```diff
@@ -44,7 +44,7 @@ RUN apt-get update \
        patch \
        ruby-dev \
     && gem install bundler --conservative \
-    && bundle install --without deploy production \
+    && bundle install --deployment --without development test \
```

Comment:
Vendoring gems in the production/base image.  It's the way we do it currently in prod, but not sure it's how we'll continue to do it in the brave new container world(?).

Reply:
Oh that's a good question, I have no idea what our goal setup is, I assumed we'd want to continue vendoring in the container world, but I guess as long as the files are saved anywhere in the filesystem and still saved in the image, we're fine?

Reply:
@pauldoomgov using these options install gems under /upaya/vendor/bundle